Cuberite failed to compile on Ubuntu 16.04 LTS Xenial

### Expected behavior
It compiles and put Cuberite in Server directory.

### Actual behavior
Throws an error when running make -j 2
```bash
Scanning dependencies of target Items
[ 38%] Building CXX object src/Items/CMakeFiles/Items.dir/ItemHandler.cpp.o
[ 38%] Building CXX object src/HTTP/CMakeFiles/HTTPServer.dir/TransferEncodingParser.cpp.o
[ 40%] Building CXX object src/HTTP/CMakeFiles/HTTPServer.dir/UrlParser.cpp.o
In file included from /home/minecraft/cuberite/src/Items/ItemHandler.cpp:39:
In file included from /home/minecraft/cuberite/src/Items/ItemMinecart.h:11:
/home/minecraft/cuberite/src/Items/../Entities/Minecart.h:55:15: error:
      'DoSetSpeed' overrides a member function but is not marked 'override'
      [-Werror,-Winconsistent-missing-override]
        virtual void DoSetSpeed(double a_SpeedX, double a_SpeedY, doubl...
                     ^
/home/minecraft/cuberite/src/./UI/../Entities/Entity.h:599:15: note: overridden
      virtual function is here
        virtual void DoSetSpeed(double a_SpeedX, double a_SpeedY, doubl...
                     ^
1 error generated.
src/Items/CMakeFiles/Items.dir/build.make:62: recipe for target 'src/Items/CMakeFiles/Items.dir/ItemHandler.cpp.o' failed
make[2]: *** [src/Items/CMakeFiles/Items.dir/ItemHandler.cpp.o] Error 1
CMakeFiles/Makefile2:1200: recipe for target 'src/Items/CMakeFiles/Items.dir/all' failed
make[1]: *** [src/Items/CMakeFiles/Items.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 40%] Linking CXX static library libHTTPServer.a
[ 40%] Built target HTTPServer
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```

### Steps to reproduce the behavior
In Ubuntu 16.06 Xenial

```bash
sudo apt-get install clang git cmake make
git clone --recursive https://github.com/cuberite/cuberite.git
cd cuberite
cmake . -DCMAKE_BUILD_TYPE=RELEASE
make -j 2
```

--- a/src/Entities/Minecart.h
+++ b/src/Entities/Minecart.h
@@ -52,7 +52,7 @@ protected:
 	bool m_bIsOnDetectorRail;
 
 	// Overwrite to enforce speed limit
-	virtual void DoSetSpeed(double a_SpeedX, double a_SpeedY, double a_SpeedZ);
+	virtual void DoSetSpeed(double a_SpeedX, double a_SpeedY, double a_SpeedZ) override;
 
 	cMinecart(ePayload a_Payload, double a_X, double a_Y, double a_Z);
 

